manager-bundle/skeleton/config/config.yaml

```
@@ -178,14 +178,28 @@ webauthn:
                 require_resident_key: true
                 user_verification: !php/const Webauthn\AuthenticatorSelectionCriteria::USER_VERIFICATION_REQUIREMENT_REQUIRED
                 resident_key: !php/const Webauthn\AuthenticatorSelectionCriteria::RESIDENT_KEY_REQUIREMENT_REQUIRED
+        contao_frontend:
+            rp:
+                name: Contao Open Source CMS
+            authenticator_selection_criteria:
+                require_resident_key: true
+                user_verification: !php/const Webauthn\AuthenticatorSelectionCriteria::USER_VERIFICATION_REQUIREMENT_REQUIRED
+                resident_key: !php/const Webauthn\AuthenticatorSelectionCriteria::RESIDENT_KEY_REQUIREMENT_REQUIRED
     request_profiles:
         contao_backend:
             user_verification: !php/const Webauthn\AuthenticatorSelectionCriteria::USER_VERIFICATION_REQUIREMENT_REQUIRED
+        contao_frontend:
+            user_verification: !php/const Webauthn\AuthenticatorSelectionCriteria::USER_VERIFICATION_REQUIREMENT_REQUIRED
     controllers:
         enabled: true
         creation:
             contao_backend_add_authenticator:
                 options_path: '%contao.backend.route_prefix%/webauthn/add/options'
                 result_path: '%contao.backend.route_prefix%/webauthn/add/result'
-                user_entity_guesser: Webauthn\Bundle\Security\Guesser\CurrentUserEntityGuesser
+                user_entity_guesser: contao.webauthn.user_entity_guesser
                 profile: contao_backend
+            contao_frontend_add_authenticator:
+                options_path: /_contao/webauthn/frontend/add/options
+                result_path: /_contao/webauthn/frontend/add/result
```

```suggestion
                options_path: /_contao/webauthn/add/options
                result_path: /_contao/webauthn/add/result
```

Because the back end paths do not have a `/backend/` fragment, either. Or should we maybe add it there instead?

---

With the current paths it does not make sense to add `/backend/`, because any path starting with `/contao` (`%contao.backend.route_prefix%`) is inherently a "backend" path of course. So either we remove it, or we use

* `/_contao/webauthn/backend/`
* `/_contao/webauthn/frontend/`

i.e. `/_contao` for the backend as well instead of `%contao.backend.route_prefix%`.

---

`_contao`  is always frontend internals to me, so I agree with @leofeyer we wouldn't need that.